### DISCUSSION TITLE
Performance improvements

### DISCUSSION THREAD

**Discussion by nikosmichas** [revsys/django-friendship#165] [General]
Hello guys,
Is there any chance to merge https://github.com/revsys/django-friendship/pull/155 and include it in a release?
I was checking some performance issues in my project (using https://github.com/jmcarp/nplusone helped me identify the cause) and some N+1 queries could be resolved with these changes. 
If there is something pending in the PR(like tests or anything else) in order to be merged, I would be glad to help.

**Comment by frankwiles** [comment]
Yep thanks for the reminder! I just merged and released them in 1.9.6.
